### PR TITLE
feat: open issue

### PR DESCRIPTION
**Pull request by doinel1a**

### CODE REVIEW

File: src/app/_components/tabs/file/vulnerability.tsx [hunk 1]
```diff
@@ -0,0 +1,141 @@
+import React, { useState } from 'react';
+
+import type { TVulnerability } from '@/agents/audit';
+
+import { CircleDot, Loader2 } from 'lucide-react';
+import Link from 'next/link';
+
+import { Button } from '@/components/ui/button';
+import { Textarea } from '@/components/ui/textarea';
```
Comment (urataps) [inline src/app/_components/tabs/file/vulnerability.tsx:9]: nit: TextArea

**Review by urataps — APPROVED** [review]
LGTM

File: src/app/_components/tabs/file/vulnerability.tsx (hunk 1 above)
Comment (doinel1a) [reply src/app/_components/tabs/file/vulnerability.tsx:9]: That's the default export from shadcn/ui. Wontfix :P

File: src/app/_components/repos/audit-dialog.tsx [hunk 2]
```diff
@@ -139,7 +139,12 @@ export default function AuditDialog({
           >
             <ul className='flex h-full w-full flex-col gap-y-5'>
               {vulnerabilitiesPromise?.map((promise, index) => (
-                <FileRow key={index} promise={promise} />
+                <FileRow
+                  key={index}
+                  ghUsername={ghUsername}
+                  repoName={repoName}
+                  promise={promise}
```
[the remark this answers is not in the document]
Comment (doinel1a) [reply src/app/_components/repos/audit-dialog.tsx]: Fixed

**Pull request merged**